app/Models/ExperienceAward.php

```diff
@@ -27,18 +28,20 @@ class ExperienceAward extends BaseModel
         'award_recipient_type_id' => 'int',
         'issued_by' => 'string',
         'award_recognition_type_id' => 'int',
-        'awarded_date' => 'date'
+        'awarded_date' => 'date',
+        'is_education_requirement' => 'boolean'
     ];
 
     protected $fillable = [
         'title',
         'award_recipient_type_id',
         'issued_by',
         'award_recognition_type_id',
-        'awarded_date'
+        'awarded_date',
+        'is_education_requirement'
     ];
 
-    protected $table = 'experiences_award';
+    protected $table = ['experiences_award'];
```

Comment:
Oh, I forgot one more comment! These table statements should be strings, not arrays (ie whats already in dev).

Reply:
I'll fix it, and merge this in

Reply:
Oh sweet, nvm.